src/styles/blocks/seconds.scss

```diff
@@ -0,0 +1,9 @@
+.seconds {
+  @include hand;
+  @include position;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.